src/components/UI/Card.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { VscEdit } from "react-icons/vsc";
+import { VscSave } from "react-icons/vsc";
+import { VscClose } from "react-icons/vsc";
+//import classnames from 'classnames';
+
+import "./Card.css";
+
+var classNames = require("classnames");
+
+class Card extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: false,
+      clicked: false,
+        input: "",
+        inputValue: "",
+        textarea: "",
+        textareaValue: "",
+    };
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.setState({ input: this.state.inputValue });
+    this.setState({ textarea: this.state.textareaValue });
+  }
+
+  inputChangeHandler(event) {
+    this.setState({ inputValue: event.target.value });
+  }
+
+  textareaChangeHandler(event) {
+    this.setState({ textareaValue: event.target.value });
+  }
```

Comment:
Again referring to the same item. Try using a common changeHandler for any property provided in a dynamic way. Please read about setting dynamic properties to objects.